crates/matrix-sdk-crypto/src/session_manager/group_sessions.rs

```diff
@@ -704,6 +697,21 @@ impl GroupSessionManager {
         result
     }
 
+    #[cfg(not(feature = "message-ids"))]
+    /// Given a to-device request, build a recipient map suitable for logging
+    ///
+    /// Returns a list of pairs of (user id, device_id)
```

Comment:
Docs before attributes, please.

```suggestion
    /// Given a to-device request, build a recipient map suitable for logging
    ///
    /// Returns a list of pairs of (user id, device_id)
    #[cfg(not(feature = "message-ids"))]
```